.travis.yml

```diff
@@ -21,6 +21,15 @@ cache:
     - $HOME/.npm
     - $HOME/.cache # includes bower's cache
 
+env:
+  - NPM_SCRIPT=test-all:cover
+
+matrix:
+  fast_finish: true
```

Comment:
Mildly opposed to `fast_finish` but since we have no "allowed_failures" this just means that the check mark gets updated a touch faster.